readthedocs/projects/forms.py

```diff
@@ -348,7 +349,7 @@ def can_build_external_versions(self, integrations):
 
     def clean_conf_py_file(self):
         filename = self.cleaned_data.get('conf_py_file', '').strip()
-        if filename and 'conf.py' not in filename:
+        if filename and not filename.endswith("conf.py"):
```

Comment:
Why are we changing this logic here? Seems out of scope for this PR, and might randomly break things.

Reply:
right, reverted :+1: